crates/pallets/starknet/src/lib.rs

```diff
@@ -238,6 +215,11 @@ pub mod pallet {
     #[pallet::getter(fn tx_events)]
     pub(super) type TxEvents<T: Config> = StorageMap<_, Identity, TransactionHash, Vec<StarknetEvent>, ValueQuery>;
 
+    #[pallet::storage]
+    #[pallet::unbounded]
+    #[pallet::getter(fn tx_messages)]
+    pub(super) type TxMessages<T: Config> = StorageMap<_, Identity, TransactionHash, Vec<MessageToL1>, ValueQuery>;
+
```

Comment:
Where does this come from? Your PR is about l1->l2 messages, why do you have to bother with the l2->l1 messages?

Reply:
I have no idea what happened here while it is not my code...

Reply:
This is also not my code...

Reply:
This code is present on the l1-l2-messaging branch